### PR TITLE
#945 update dags folder readme.md

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:

A minor shuffling of dags docs as requested by @chmnata!

- Add description of files/folders in dags/ (including the functions in common_tasks, dag_functions)
- Add TOC
- Replace etc/airflow with data/airflow
- Reduce some duplication between "add a new DAG" and "update an existing DAG" sections

## Issue(s) this solves:

- #945 

## What, in particular, needs to reviewed:

- File structure

### CODE REVIEW

File: dags/readme.md [hunk 1]
```diff
@@ -1,20 +1,116 @@
...
+- Deprecated: [pull_weather.py](pull_weather.py)
+
+### Misc
+- [**log_cleanup.py**](log_cleanup.py): A maintenance workflow to periodically clean out the Airflow task logs to avoid those getting too big.
+- [eoy_create_tables.py](eoy_create_tables.py): An end of year maintenance DAG which creates new partitions and maintains calendar tables.  
+
+### Here
+- **[pull_here.py](pull_here.py)**: [readme](../here/traffic/README.md#probe_path).  
+- **[pull_here_path.py](pull_here_path.py)**: [readme](../here/traffic/README.md#path).  
+- HERE Aggregations: [citywide_tti_aggregate.py](citywide_tti_aggregate.py), [traffic_transfer.py](traffic_transfer.py).
```
Comment (chmnata) [inline dags/readme.md:47]: traffic_transfer is not a here aggregation, seems like its doing some transfer stuff in the `traffic` schema?

File: dags/readme.md [hunk 2]
```diff
@@ -52,119 +148,36 @@ ln -sf <PATH-TO-DAG-ON-YOUR-HOME-FOLDER> airflow/dags/<DAG-FILE>
...
 
 10) Head back to GitHub and make a pull request for your pipeline branch to master. Once your pull request is approved, a sys admin will handle the rest so please be sure to specify anything that needs to happen to get the DAG running. It may include things such as:
 - Changing references to personal schemas in any supporting sql scripts
-- Moving the DAG from your personal home folder to `/etc/airflow/data_scripts`
+- Moving the DAG from your personal home folder to `/data/airflow/data_scripts`
 - Mentioning any DAGs that this DAG depends on, or other DAGs that depend on this DAG
 
-11) **For sys admins only:** Once the PR is approved and merged, complete all of the sys admin tasks listed in the PR. Then, navigate to `/etc/airflow/data_scripts` and git pull from master using:
+11) **For sys admins only:** Once the PR is approved and merged, complete all of the sys admin tasks listed in the PR. Then, navigate to `/data/airflow/data_scripts` and git pull from master using:
```
Comment (chmnata) [inline dags/readme.md:160]: Can we change sys admins -> airflow user ?

File: dags/readme.md [hunk 3]
```diff
@@ -1,20 +1,116 @@
...
+### VDS (Formerly RESCU)
+- **[vds_check.py](vds_check.py)**: [readme](../volumes/vds/readme.md#vds_check-dag).  
+- **[vds_pull_vdsdata.py](vds_pull_vdsdata.py)**: [readme](../volumes/vds/readme.md#vds_pull_vdsdata-dag).  
+- **[vds_pull_vdsvehicledata.py](vds_pull_vdsvehicledata.py)**: [readme](../volumes/vds/readme.md#vds_pull_vdsvehicledata-dag).  
+- Deprecated: [check_rescu.py](check_rescu.py). 
+
+### WYS
+- [**refresh_wys_monthly.py**](refresh_wys_monthly.py): [readme](../wys/api/README.md#wys_monthly_summary).
+- [**pull_wys.py**](pull_wys.py): [readme](../wys/api/README.md#pull_wys).
+- [**wys_check.py**](wys_check.py): Contains additional data quality checks for `pull_wys`.
```
Comment (chmnata) [inline dags/readme.md:102]: ohhhhhhh we dont have a readme section on this 👁️ 👁️

**Review by chmnata — CHANGES_REQUESTED** [review]
smol changes plz

File: dags/readme.md (hunk 1 above)
Comment (gabrielwol) [reply dags/readme.md:47]: Ah yes it looks like an outdated part of the replication pipeline.

File: dags/readme.md (hunk 2 above)
Comment (chmnata) [reply dags/readme.md:160]: since everyone with airflow account access can git pull 👀

**Review by chmnata — APPROVED** [review]
:gabe-approved: thanks for updating this!

**Pull request merged**